src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_CrustalInversionRunner.java

```diff
@@ -40,13 +41,25 @@
     private double sansSlipRateFactor = -1;
     private double tvzSlipRateFactor = -1;
 
+    private boolean enableTvz = true;
+
     /**
      * Creates a new NZSHM22_InversionRunner with defaults.
      */
     public NZSHM22_CrustalInversionRunner() {
         super();
     }
 
+    /**
+     * Determines whether NZ is a single region (false) or is split up in TVZ and Sans TVZ (true)
+     * @param enableTvz
+     * @return
+     */
+    public NZSHM22_CrustalInversionRunner setEnableTvz(boolean enableTvz){
```

Comment:
I suggest we say this is an MFD  only setting - eg `setEnableTvzMFDs`

Reply:
done